vendor.conf

```diff
@@ -97,3 +97,4 @@ github.com/openshift/imagebuilder master
 github.com/ulikunitz/xz v0.5.4
 github.com/mailru/easyjson 03f2033d19d5860aef995fe360ac7d395cd8ce65
 github.com/coreos/go-iptables 25d087f3cffd9aedc0c2b7eff25f23cbf3c20fe1
+github.com/docker/libnetwork 20461b8539336a4b5fcf551a86dd24ebae211984
```

Comment:
I think this can be removed given the code went into a dedicated package.

Reply:
Good call, fixed